satellite/audit/service.go

```diff
@@ -41,12 +44,18 @@ type Service struct {
 	Reporter reporter
 
 	Loop sync2.Cycle
+
+	// for audit 2.0 using metainfoloop
+	ReservoirLoop  sync2.Cycle
```

Comment:
Is there a reason it's not implementing a new service instead?

Reply:
now it's been separated into audit.Service and audit.ReservoirService